src/vm/jitinterface.cpp

```diff
@@ -6991,7 +6991,7 @@ bool getILIntrinsicImplementationForUnsafe(MethodDesc * ftn,
     else if (tk == MscorlibBinder::GetMethod(METHOD__UNSAFE__BYREF_AS)->GetMemberDef())
     {
         // Return the argument that was passed in.
-        static const BYTE ilcode[] = { CEE_LDARG_0, CEE_RET };
+        static const BYTE ilcode[] = { CEE_LDARG_0, CEE_STLOC_0, CEE_LDLOC_0, CEE_RET };
```

Comment:
I ran build at root of coreclr repo. How else should I verify that the change is correct?